### DISCUSSION TITLE
WLED too warm color

### DISCUSSION THREAD

**Discussion by sylvaindc** [basnijholt/adaptive-lighting#809] [Q&A]
Hi,

Adaptive lightning works extremely well for my HUE bulbs, but the WLED controlled strips I use (RGBW) are way off on the warm side (the light is too yellow). When I crank up the temp in adaptive lightning config to 10k it moves in the right directly, but its stilll too yellow (this probably due to the fact that I'm using WarmWhite LEDs instead of ColdWhite ones).

Is there any way to adjust for this, by e.g. setting a higher temp multiplier?

**Comment by BadRockk** [comment]
Hi, 
did you found a solution to make it work properly?

**Reply by sylvaindc** [reply]
No, sadly I didnt. But to be honest I didnt spend too much time on it either.  
Do you have a similar issue?

**Reply by BadRockk** [reply]
Yes, with SK6812 (RGBW) stripes. With my Aqara TM1 it works perfectly but with the SK6812 I get red light in the evening.

In WLED I see 3 parameters: brightness for RGB, brightness and temperature for white channel. I think adaptive lighting can’t handle those together at once.

**Reply by hogantg** [reply]
This is the same issue I am having with SK6812 RGBW (NW). >3500kelvin works fine, but any lower and the lights look red. 

I have actually been running the "warm" (<3500k) colors on the strip by RGB value before trying to use adaptive lighting. To be honest, it works just fine for me. I am disappointed that the AL setting (use RGB) does not work correctly. Especially when I have set the "sleep mode" to use RGB with great success at matching a warm 2500k tone.

Trying to find solutions. Have not made any changes but these are the things im going to play with when i have time:

1. WLED's "White handling" setting
2. Turn of AL "adapt color temperature" and just set the color temps on my own throughout the day
3. Modify the code as seen here: #226 
4. Make a [color temp light blueprint](https://community.home-assistant.io/t/use-a-rgbw-light-as-a-light-with-color-temperature-control/379484) from my light entity

**Reply by sneak-it** [reply]
Chiming in to say same issue here, WLED with a SK6812 RGBNW strip. The 'warm' Adaptive Lighting chooses is more of a pinkish red which doesn't match most realistic lighting conditions.
